include/pando-lib-galois/graphs/dist_local_csr.hpp

```diff
@@ -693,6 +728,62 @@ class DistLocalCSR {
     return pando::Status::Success;
   }
 
+  /**
+   * @brief This function creates a mirror list for each host. Currently it implements full
+   * mirroring
+   */
+  template <typename ReadEdgeType>
+  HostLocalStorage<pando::Array<std::uint64_t>> getMirrorList(
+      galois::HostIndexedMap<pando::Vector<pando::Vector<ReadEdgeType>>> partEdges,
+      HostLocalStorage<pando::Array<std::uint64_t>> V2PM) {
+    HostLocalStorage<pando::Array<std::uint64_t>> mirrorList;
+    PANDO_CHECK(mirrorList.initialize());
+    auto createMirrors =
+        +[](galois::HostIndexedMap<pando::Vector<pando::Vector<ReadEdgeType>>> partEdges,
+            HostLocalStorage<pando::Array<std::uint64_t>> mirrorList,
+            HostLocalStorage<pando::Array<std::uint64_t>> V2PM, std::uint64_t i,
+            galois::WaitGroup::HandleType wgh) {
+          pando::Array<uint64_t> mirrors;
+
+          // Populating the mirror list in a set to avoid duplicates
+          std::unordered_set<uint64_t> mirrorMap;
```

Comment:
This data structure cannot be used. instead use a hashtable to a boolean.

Reply:
@jeageun This is your code I think

Reply:
This one is only for one construction. 
Also, how could we know about the  address of target using boolean? 
@divija95